src/messages.ts

```diff
@@ -456,32 +456,60 @@ export const bottomMenuMessages = defineMessages('ui.src.bottomMenu', {
   },
 });
 
+export const purposeMessages = defineMessages('purpose.trackingType', {
+  'essential.title': {
+    defaultMessage: 'Essential',
+    description: 'Text for essential purposes in CompleteOptions view state.',
+  },
+  'essential.description': {
+    defaultMessage: 'No consent needed.',
```

Comment:
this didnt have a description before, right? we should leave this empty

Reply:
I think we want to show description for Essential too: https://transcend.height.app/T-38707#activityId=00cf11bc-9d8d-4ba9-853e-493bde392e3a

Reply:
oh cool! didnt see the ask, nvm then